server/schedulers/utils.go

```diff
@@ -197,6 +197,8 @@ func newPendingInfluence(op *operator.Operator, from, to uint64, infl Influence)
 	}
 }
 
+// summaryPendingInfluence calculate the summary pending Influence for each store and return storeID-> Influence
+// It make each rate(key/byte/count) become (1+w) times to the origin value while f is the function to provide w(weight)
```

Comment:
```suggestion
// It makes each rate(key/byte/count) become (1+w) times to the origin value while f is the function to provide w(weight)
```

And there is no count rate.